app/components/orange-bar-custom-map-study.js

```diff
@@ -0,0 +1,165 @@
+import Component from '@ember/component';
+import { alias } from '@ember/object/computed';
+import { inject as service } from '@ember/service';
+import { computed } from '@ember/object';
+import { task } from 'ember-concurrency';
+import numeral from 'numeral';
+import fetch from 'fetch';
+import Environment from '../config/environment';
+import choroplethConfigs from '../choropleth-config';
+
+const { SupportServiceHost } = Environment;
+
+export default Component.extend({
```

Comment:
I'm assuming this file is the combination of other code elsewhere

Reply:
Completely taken from map-utility-box.js